wpiutil/src/main/native/windows/StackTrace.cpp

```diff
@@ -8,6 +8,8 @@
 #include "wpi/ConvertUTF.h"
 #include "wpi/SmallString.h"
 
+#if defined(_MSC_VER)
```

Comment:
If we’re doing it with ifdefs, this should be checking for _WIN32, not MSCVER.

Reply:
StackWalker.h uses _MSC_VER.

Reply:
Ah.  Makes sense for us to be consistent then, and maybe a broader cleanup is worth looking at in a separate PR.